packages/frontend/src/components/chat/ChatList.tsx

```diff
@@ -856,7 +836,15 @@ export function useLogicVirtualChatList(chatListIds: number[]) {
     }
   }, [accountId])
 
-  return { isChatLoaded, loadChats, chatCache }
+  return {
+    /**
+     * This function is not reactive, i.e. it doesn't get re-created
+     * when we start loading more items.
```

Comment:
```suggestion
     * when we start loading more items.  It also returns true if a
     * chat is not yet loaded but is in the process of being loaded.
```

Reply:
I suggest to add this comment or rename the function since the current name could be misleading

Reply:
Can be done in a follow-up